### PR TITLE
Responsive top nav

### PR DESCRIPTION
**Pull request by admrid**
#### Trello card / screenshot:
- https://trello.com/c/VmpI9KVT
- [img01](https://trello-attachments.s3.amazonaws.com/5a745f7bf15cbea9787c0c54/5b3109acf19129eb94bcece5/17c72da3fa607fd9865191946414e803/image.png)
- [img02](https://trello-attachments.s3.amazonaws.com/5a745f7bf15cbea9787c0c54/5b3109acf19129eb94bcece5/d6ff60a214500e9c68c4c7bce607abf9/image.png)

#### Is connection necessary to test? If so which network?
- [ ] local RPC
- [x] Rinkeby
- [ ] Main Ethereum Network

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 9bc277febf5bee9d97fd6afb5d6647346f7974db

https://app.netlify.com/sites/augmint/deploys/5b3f8cd8fdd72a7aebe4656d

File: src/components/accountInfo/styles.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+export const StyledAccountP = styled.p`
+    font-weight: bolder;
+
+    &.accInfoDetail {
+        display: none;
+    }
+
+    ${media.tablet`
+        color: ${theme.colors.primary};
+        font-size: 20px;
```
Comment (reyrodrigez) [inline src/components/accountInfo/styles.js]: please use the `remCalc()` helper! +next line as well

File: src/components/accountInfo/styles.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            font-size: 20px;
+            line-height: 26px;
+        }
+    `};
+`;
+
+export const StyledAccountInfo = styled(Pblock)`
+    &.accountInfo {
+        padding: 0px 10px;
+        border: 1px solid #ffad00;
```
Comment (reyrodrigez) [inline src/components/accountInfo/styles.js]: do we have `#ffad00` in `theme.colors`?

File: src/components/augmint-ui/header/styles.js [hunk 3]
```diff
@@ -53,7 +54,8 @@ const HeaderH2 = `
 `;
 
 const HeaderH1 = `
-    font-size: ${theme.typography.fontSizes.h1};
+    // font-size: ${theme.typography.fontSizes.h1};
+    font-size: 2rem;
```
Comment (reyrodrigez) [inline src/components/augmint-ui/header/styles.js]: use `remCalc()` pls

File: src/components/dashboard/containers/topNav/index.js [hunk 4]
```diff
@@ -16,47 +17,84 @@ import {
...
                     </StyledTopNavLi>
-                    <StyledTopNavLi className="account">
-                        <StyledTopNavLinkRight title="Your account" to="/account" className="accountDetails">
-                            <Icon name="account" className="accountIcon" />
-                            <StyledPrice>
+                    <StyledTopNavLi className={this.props.showAccInfo ? "account" : "account hidden"}>
+                        <StyledTopNavLinkRight
+                            title="Your account"
+                            to={this.props.showAccInfo ? "" : "/account"}
+                            onTouchStart={this.toggleAccInfo}
```
Comment (reyrodrigez) [inline src/components/dashboard/containers/topNav/index.js]: are you sure you need the `onTouchStart` event handler? Touch devices are usually understands the notion of `onClick`

**Comment by admrid** [comment]
@reyrodrigez yes but it works on the actual devices :S

**Comment by admrid** [comment]
thanks @szerintedmi & @reyrodrigez , will look into it on monday.
it works on iOS (both ipad & iphone).
i might not need both touch & click. there was a time when i def needed it. 
now it has to work on mobile, and on tablet size on dektop and it has to work on desktop as a hover as well when it's media is < than tablet. i take a look at it again anyway.

File: src/containers/app/index.js [hunk 5]
```diff
@@ -119,22 +129,22 @@ class App extends React.Component {
...
                         showMenu={this.state.showMobileMenu}
                         toggleMenu={this.toggleMenu}
                     />
-                }
-                {showConnection 
-                    ? <SideNav showMenu={this.state.showMobileMenu} toggleMenu={this.toggleMenu} /> 
-                    : null
-                }
+                )}
+                {showConnection ? <SideNav showMenu={this.state.showMobileMenu} toggleMenu={this.toggleMenu} /> : null}
```
Comment (gyurka00) [inline src/containers/app/index.js]: please do not use ternary operator if it is not necessary.

`{showConnection && <SideNav showMenu={this.state.showMobileMenu} toggleMenu={this.toggleMenu} />}`

**Pull request merged**